vassal-app/src/main/java/VASSAL/counters/PieceDefiner.java

```diff
@@ -1295,33 +1295,29 @@ private static String cellText(int lineno, String s) {
 
     @Override
     public Component getListCellRendererComponent(
-      JList list, Object value, int index, boolean selected, boolean hasFocus) {
+            JList list, Object value, int index, boolean selected, boolean hasFocus) {
 
       // DO NOT pass value to super.getListCellRendererComponent()
       // It is incredibly inefficient for GamePieces and is not needed
       // since we overwrite the label text anyway.
       super.getListCellRendererComponent(list, "", index, selected, hasFocus);
 
-      final int lineOffset = definer.isPrototype() ? 1 : 0;
-      final int lineno = index + lineOffset;
+      // For Pieces, bump index to 1
+      final int lineno = definer.isPrototype() ? index + 1 : index;
 
       if (value instanceof EditablePiece) {
-        final EditablePiece editableValue = (EditablePiece) value;
-
-        // Prototype? Skip first (dummy) item to avoid outputting a blank line
-        if (lineOffset == 1 || index > 0) {
-          // For Pieces, bump index to 1
-          // Special formatting for Comments
-          if ((editableValue.getType().startsWith(Comment.ID))) {
-            setText(cellText(lineno, "<b>/* " + ConfigureTree.noHTML((editableValue.getDescription())) + " */</b>"));
-          }
-          else {
-            setText(cellText(lineno, ConfigureTree.noHTML(editableValue.getDescription())));
-          }
+
+        if (value instanceof Decorator && ((Decorator) value).myGetType().startsWith(Comment.ID)) {
```

Comment:
You should be able to replace this with just

`if (value instanceof Comment) {`

which will be more efficient.

Reply:
Change made in PR #13001.